toolchain/check/testdata/class/fail_method_modifiers.carbon

```diff
@@ -0,0 +1,103 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+class FinalClass {
+  // CHECK:STDERR: fail_method_modifiers.carbon:[[@LINE+9]]:3: ERROR: Semantics TODO: `private`.
+  // CHECK:STDERR:   private abstract fn Abstract[self: Self]();
+  // CHECK:STDERR:   ^~~~~~~
+  // CHECK:STDERR: fail_method_modifiers.carbon:[[@LINE+6]]:11: ERROR: `abstract` not allowed on `fn` declaration in a non-abstract `class` definition.
+  // CHECK:STDERR:   private abstract fn Abstract[self: Self]();
+  // CHECK:STDERR:           ^~~~~~~~
+  // CHECK:STDERR: fail_method_modifiers.carbon:[[@LINE-7]]:1: Containing definition here.
+  // CHECK:STDERR: class FinalClass {
+  // CHECK:STDERR: ^~~~~
+  private abstract fn Abstract[self: Self]();
+  // CHECK:STDERR: fail_method_modifiers.carbon:[[@LINE+9]]:3: ERROR: Semantics TODO: `protected`.
```

Comment:
```suggestion
  private abstract fn Abstract[self: Self]();

  // CHECK:STDERR: fail_method_modifiers.carbon:[[@LINE+9]]:3: ERROR: Semantics TODO: `protected`.
```

In these class tests, you seem to have put no blank lines between members. This is making it hard for me to identify where there's a line of code in the middle of blocks of output, since output also includes code.

What would you think of adding blank lines between lines of code in the various tests?